frontend/src/components/TerminalTarget.vue

```diff
@@ -69,14 +72,19 @@ export default {
       type: Object
     }
   },
-  mixins: [shootItem],
   computed: {
     ...mapGetters([
       'hasControlPlaneTerminalAccess',
       'hasShootTerminalAccess',
       'hasGardenTerminalAccess',
       'isAdmin'
     ]),
+    shootName () {
+      return get(this.shootItem, 'metadata.name')
+    },
+    isShootStatusHibernated () {
+      return isShootStatusHibernated(get(this.shootItem, 'status'))
+    },
```

Comment:
The only two methods used from the mixin. The `shootItem` property can be empty in this case. In the mixin I have defined it as required.